.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  "packages/libp2p":"0.45.4"
```

Comment:
Should this be `0.45.5` ?

Reply:
Yeah, it's a pain keeping this PR up to date.